cmd/podman/ps.go

```diff
@@ -413,8 +413,15 @@ func getTemplateOutput(containers []*libpod.Container, opts psOptions) ([]psTemp
 		if err != nil {
 			return psOutput, errors.Wrapf(err, "unable to obtain container pid")
 		}
-		runningFor := units.HumanDuration(time.Since(conConfig.CreatedTime))
-		createdAt := runningFor + " ago"
+		startedTime, err := ctr.StartedTime()
+		if err != nil {
+			logrus.Errorf("error getting started time for %q: %v", ctr.ID(), err)
+		}
+		runningFor := units.HumanDuration(time.Since(startedTime))
+		if startedTime == (time.Time{}) {
+			runningFor = ""
+		}
+		createdAt := conConfig.CreatedTime.Format("2006-01-02 15:04:05 -0700 MST")
```

Comment:
This is fine, just wondering if this might be a little clearer:

```
runningFor := ""
startedTime, err := ctr.StartedTime()
if err != nil {
	logrus.Errorf("error getting started time for %q: %v", ctr.ID(), err)
} else {
	runningFor = units.HumanDuration(time.Since(startedTime))
}
```

Reply:
Reformatted it a bit to make it clearer.